collections/nemo_asr/nemo_asr/helpers.py

```diff
@@ -28,7 +28,9 @@ def __ctc_decoder_predictions_tensor(tensor, labels):
     return hypotheses
 
 
-def monitor_asr_train_progress(tensors: list, labels: list, tb_logger=None,
+def monitor_asr_train_progress(tensors: list, labels: list,
+                               use_wer=True,
```

Comment:
Suggestion: instead of `use_wer`, maybe this should be a parameter such as `eval_metric` that takes string arguments where the options are 'WER' and 'CER' (plus an appropriate check in function body for argument validity) where the default is WER. That should make it clearer what is being done when looking at a call to this function.

Reply:
(Similar suggestion for `process_evaluation_epoch`.)